src/lib/transition_handler/processor.ml

```diff
@@ -0,0 +1,29 @@
+open Protocols.Coda_pow
+
+module Make (Inputs : Inputs_intf.S) : Transaction_handler_processor_intf = struct
+  open Inputs
+
+  (* TODO: implement drop old *)
```

Comment:
This is the `force_write_maybe_drop_head` function on `Linear_pipe` (we should rename it)

Reply:
Ahh, wasn't aware of that function. Thought I would have to implement it myself.